src/status_im2/common/bottom_sheet/events.cljs

```diff
@@ -3,7 +3,7 @@
 
 (rf/defn show-bottom-sheet
   [{:keys [db]} {:keys [view options]}]
-  {:dispatch                  [:dismiss-keyboard]
+  {:dispatch                  [[:dismiss-keyboard]]
```

Comment:
is this needed?

Reply:
well, yep, I think it is although it has to be changed to `:dispatch-n`.

The problem is that when you merge this with other effects some of them might already contain `:dispatch` and this will cause some problems (it did cause them with keycard code). `:dispatch-n` is safer in this case I suppose

Reply:
coool.